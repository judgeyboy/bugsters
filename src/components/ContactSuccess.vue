<template>
  <div class="order-success">
    <div class="empty">
      <div class="empty-icon">
        <i class="icon icon-3x icon-check"></i>
      </div>
      <p class="empty-title h5">Thank you for getting in touch</p>
      <p class="empty-subtitle">We'll get back to you as soon as possible</p>
      <div class="empty-action">
        <a @click="close">Close</a>
      </div>
    </div>
  </div>
</template>

<script>
export default {
  name: 'ContactSuccess',
  methods: {
    close () {
      this.$emit('closeSuccess')
    }
  }
}
</script>
