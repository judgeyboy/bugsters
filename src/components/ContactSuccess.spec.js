import { shallow } from 'vue-test-utils'

import ContactSuccess from './ContactSuccess'

describe('ContactSuccess Component', () => {

  it('should be a Vue instance', () => {
    const wrapper = shallow(ContactSuccess)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(ContactSuccess)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
