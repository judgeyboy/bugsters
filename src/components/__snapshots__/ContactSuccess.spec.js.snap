// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactSuccess Component should match snapshot 1`] = `
"<div class=\\"order-success\\"><h2 class=\\"text-center\\">Thank you :)</h2> <p>
    Great! We've received your message.
    We'll get back to you as soon as possible
  </p></div>"
`;
